src/open_inwoner/configurations/bootstrap/constants.py

```diff
@@ -0,0 +1,229 @@
+from mozilla_django_oidc_db.models import UserInformationClaimsSources
+from zgw_consumers.api_models.constants import VertrouwelijkheidsAanduidingen
+
+from open_inwoner.openzaak.models import generate_default_file_extensions
+
+from .utils import generate_api_fields_from_template
+
+CONFIDENTIALITY_CHOICES = [
+    choice[0] for choice in VertrouwelijkheidsAanduidingen.choices
+]
+FILE_EXTENSION_CHOICES = [ext for ext in generate_default_file_extensions()]
+USER_INFO_CLAIM_SOURCES_CHOICES = [
+    choice[0] for choice in UserInformationClaimsSources.choices
+]
```

Comment:
'Choices' as naming convention in Django means a value and a label. If it is just a set of single values then call it something else.